### ISSUE TITLE
EchoStar SAGE206612 Doorbell Sensor -  Error in Domoticz log when adding

### ISSUE THREAD

**Issue by dognutz** [stas-demydiuk/domoticz-zigbee2mqtt-plugin#573]
**Issue description**
Errors when trying to add device. Device shows up in Zigbee2MQTT but Domoticz cannot add device.
Please let me know any other info that would be useful, thanks.

**Additional information**
Zigbee2MQTT version: Latest
Python version: 3.7.?
Domoticz version: Latest
Plugin version: Latest (updated an hour before this post)

**Logs**
2021-02-10 15:07:09.684 Error: (Zigbee2MQTT) 0x0017880106a30bc3: can not process numeric item "color_temp_startup"
2021-02-10 15:07:09.685 Error: (Zigbee2MQTT) 0x0017880106a30bc3: can not process feature type "composite"
2021-02-10 15:07:09.686 Error: Expected: myVar = Domoticz.Device(Name="myDevice", Unit=0, TypeName="", Type=0, Subtype=0, Switchtype=0, Image=0, Options={}, Used=1)
2021-02-10 15:07:09.686 Error: (Zigbee2MQTT) 'CDevice_init' failed 'TypeError':'an integer is required (got type NoneType)'.
2021-02-10 15:07:09.686 Error: Device creation failed, Device object is not associated with a plugin.
2021-02-10 15:07:09.687 Error: (Zigbee2MQTT) 0x00124b00222e14f5: can not process numeric item "color_temp_startup"
2021-02-10 15:07:09.688 Error: Expected: myVar = Domoticz.Device(Name="myDevice", Unit=0, TypeName="", Type=0, Subtype=0, Switchtype=0, Image=0, Options={}, Used=1)
2021-02-10 15:07:09.688 Error: (Zigbee2MQTT) 'CDevice_init' failed 'TypeError':'an integer is required (got type NoneType)'.
2021-02-10 15:07:09.688 Error: Device creation failed, Device object is not associated with a plugin.
2021-02-10 15:07:09.690 Error: Expected: myVar = Domoticz.Device(Name="myDevice", Unit=0, TypeName="", Type=0, Subtype=0, Switchtype=0, Image=0, Options={}, Used=1)
2021-02-10 15:07:09.690 Error: (Zigbee2MQTT) 'CDevice_init' failed 'TypeError':'an integer is required (got type NoneType)'.
2021-02-10 15:07:09.690 Error: Device creation failed, Device object is not associated with a plugin.
2021-02-10 15:07:09.691 Error: Expected: myVar = Domoticz.Device(Name="myDevice", Unit=0, TypeName="", Type=0, Subtype=0, Switchtype=0, Image=0, Options={}, Used=1)
2021-02-10 15:07:09.691 Error: (Zigbee2MQTT) 'CDevice_init' failed 'TypeError':'an integer is required (got type NoneType)'.
2021-02-10 15:07:09.691 Error: Device creation failed, Device object is not associated with a plugin.
2021-02-10 15:07:09.691 Error: Expected: myVar = Domoticz.Device(Name="myDevice", Unit=0, TypeName="", Type=0, Subtype=0, Switchtype=0, Image=0, Options={}, Used=1)
2021-02-10 15:07:09.691 Error: (Zigbee2MQTT) 'CDevice_init' failed 'TypeError':'an integer is required (got type NoneType)'.
2021-02-10 15:07:09.691 Error: Device creation failed, Device object is not associated with a plugin.

**Comment by stas-demydiuk** [comment]
Could you please enable debug mode in the plugin and post here the log

**Comment by dognutz** [comment]
Sure. The gaps in the log below are where I've removed log entries from other hardware.
Thanks.
Also, probably unrelated and perhaps you'd prefer I opened a new issue, but after updating yesterday I got an error about unable to load plugin.py (and referencing AV201021.py) so I commented out the line referencing AV201021.py in domoticz/plugins/zigbee2mqtt/adapters/bitron/__init__.py
That allowed the plugin to start.
Anyway, here are the logs:


2021-02-11 08:29:25.181 (Zigbee2MQTT) Zigbee2mqtt bridge is online
2021-02-11 08:29:25.181 (Zigbee2MQTT) MqttClient::publish zigbee2mqtt/bridge/config/groups ()
2021-02-11 08:29:25.189 (Zigbee2MQTT) Received available devices list from bridge
2021-02-11 08:29:25.189 (Zigbee2MQTT) Coordinator address is 0x00124b00216000b9
2021-02-11 08:29:25.219 (Zigbee2MQTT) {"access": 7, "description": "Color temperature after cold power on of this light", "name": "color_temp_startup", "presets": [{"description": "Coolest temperature supported", "name": "coolest", "value": 150}, {"description": "Cool temperature (250 mireds / 4000 Kelvin)", "name": "cool", "value": 250}, {"description": "Neutral temperature (370 mireds / 2700 Kelvin)", "name": "neutral", "value": 370}, {"description": "Warm temperature (454 mireds / 2200 Kelvin)", "name": "warm", "value": 454}, {"description": "Warmest temperature supported", "name": "warmest", "value": 500}, {"description": "Restore previous color_temp on cold power on", "name": "previous", "value": 65535}], "property": "color_temp_startup", "type": "numeric", "unit": "mired", "value_max": 500, "value_min": 150}
2021-02-11 08:29:25.219 (Zigbee2MQTT) {"description": "Color of this light expressed as hue/saturation", "features": [{"access": 7, "name": "hue", "property": "hue", "type": "numeric"}, {"access": 7, "name": "saturation", "property": "saturation", "type": "numeric"}], "name": "color_hs", "property": "color", "type": "composite"}
2021-02-11 08:29:25.220 (Zigbee2MQTT) Creating domoticz device to handle "effect" key for device with ieeeAddr 0x0017880106a30bc3
2021-02-11 08:29:25.221 (Zigbee2MQTT) {"access": 7, "description": "Color temperature after cold power on of this light", "name": "color_temp_startup", "presets": [{"description": "Coolest temperature supported", "name": "coolest", "value": 150}, {"description": "Cool temperature (250 mireds / 4000 Kelvin)", "name": "cool", "value": 250}, {"description": "Neutral temperature (370 mireds / 2700 Kelvin)", "name": "neutral", "value": 370}, {"description": "Warm temperature (454 mireds / 2200 Kelvin)", "name": "warm", "value": 454}, {"description": "Warmest temperature supported", "name": "warmest", "value": 500}, {"description": "Restore previous color_temp on cold power on", "name": "previous", "value": 65535}], "property": "color_temp_startup", "type": "numeric", "unit": "mired", "value_max": 500, "value_min": 150}
2021-02-11 08:29:25.221 (Zigbee2MQTT) Creating domoticz device to handle "effect" key for device with ieeeAddr 0x00124b00222e14f5
2021-02-11 08:29:25.222 (Zigbee2MQTT) MqttClient::subscribe
2021-02-11 08:29:25.222 (Zigbee2MQTT) Zigbee2mqtt log level is debug
2021-02-11 08:29:25.222 (Zigbee2MQTT) Joining new devices is disabled on the zigbee bridge
2021-02-11 08:29:25.275 (Zigbee2MQTT) onMQTTSubscribed

2021-02-11 08:29:25.219 Error: (Zigbee2MQTT) 0x0017880106a30bc3: can not process numeric item "color_temp_startup"
2021-02-11 08:29:25.219 Error: (Zigbee2MQTT) 0x0017880106a30bc3: can not process feature type "composite"
2021-02-11 08:29:25.220 Error: Expected: myVar = Domoticz.Device(Name="myDevice", Unit=0, TypeName="", Type=0, Subtype=0, Switchtype=0, Image=0, Options={}, Used=1)
2021-02-11 08:29:25.220 Error: (Zigbee2MQTT) 'CDevice_init' failed 'TypeError':'an integer is required (got type NoneType)'.
2021-02-11 08:29:25.220 Error: Device creation failed, Device object is not associated with a plugin.
2021-02-11 08:29:25.221 Error: (Zigbee2MQTT) 0x00124b00222e14f5: can not process numeric item "color_temp_startup"
2021-02-11 08:29:25.222 Error: Expected: myVar = Domoticz.Device(Name="myDevice", Unit=0, TypeName="", Type=0, Subtype=0, Switchtype=0, Image=0, Options={}, Used=1)
2021-02-11 08:29:25.222 Error: (Zigbee2MQTT) 'CDevice_init' failed 'TypeError':'an integer is required (got type NoneType)'.
2021-02-11 08:29:25.222 Error: Device creation failed, Device object is not associated with a plugin.
2021-02-11 08:29:26.257 (Zigbee2MQTT) Unable to find selector switch level for value "heat", device: Zigbee2MQTT - ZB-TRV3 - KitchenTRV (Mode)
2021-02-11 08:29:26.257 (Zigbee2MQTT) Unable to find selector switch level for value "heat", device: Zigbee2MQTT - ZB-TRV3 - KitchenTRV (Mode)
2021-02-11 08:29:26.289 (Zigbee2MQTT) Unable to find selector switch level for value "heat", device: Zigbee2MQTT - ZB-TRV6 - Bedroom (door) (Mode)
2021-02-11 08:29:26.289 (Zigbee2MQTT) Unable to find selector switch level for value "heat", device: Zigbee2MQTT - ZB-TRV6 - Bedroom (door) (Mode)
2021-02-11 08:29:26.311 (Zigbee2MQTT) Unable to find selector switch level for value "heat", device: Zigbee2MQTT - ZB-TRV8 - LivingRoomTRV (Mode)
2021-02-11 08:29:26.311 (Zigbee2MQTT) Unable to find selector switch level for value "heat", device: Zigbee2MQTT - ZB-TRV8 - LivingRoomTRV (Mode)
2021-02-11 08:29:26.323 (Zigbee2MQTT) Unable to find selector switch level for value "wakeup", device: Smart Cube - Action
2021-02-11 08:29:26.323 (Zigbee2MQTT) Unable to find selector switch level for value "wakeup", device: Smart Cube - Action
2021-02-11 08:29:26.340 (Zigbee2MQTT) Creating domoticz device to handle "effect" key for device with ieeeAddr 0x0017880106a30bc3
2021-02-11 08:29:26.356 (Zigbee2MQTT) Creating domoticz device to handle "effect" key for device with ieeeAddr 0x00124b00222e14f5
2021-02-11 08:29:26.358 (Zigbee2MQTT) Received groups list from bridge
2021-02-11 08:29:26.341 Error: Expected: myVar = Domoticz.Device(Name="myDevice", Unit=0, TypeName="", Type=0, Subtype=0, Switchtype=0, Image=0, Options={}, Used=1)
2021-02-11 08:29:26.341 Error: (Zigbee2MQTT) 'CDevice_init' failed 'TypeError':'an integer is required (got type NoneType)'.
2021-02-11 08:29:26.341 Error: Device creation failed, Device object is not associated with a plugin.
2021-02-11 08:29:26.356 Error: Expected: myVar = Domoticz.Device(Name="myDevice", Unit=0, TypeName="", Type=0, Subtype=0, Switchtype=0, Image=0, Options={}, Used=1)
2021-02-11 08:29:26.356 Error: (Zigbee2MQTT) 'CDevice_init' failed 'TypeError':'an integer is required (got type NoneType)'.
2021-02-11 08:29:26.356 Error: Device creation failed, Device object is not associated with a plugin.

2021-02-11 08:29:31.769 (Zigbee2MQTT) New request: [1] bridge_getstatus({})
2021-02-11 08:29:31.769 (Zigbee2MQTT) MqttClient::publish zigbee2mqtt/bridge/config/permit_join ()
2021-02-11 08:29:32.181 (Zigbee2MQTT) Zigbee2mqtt log level is debug
2021-02-11 08:29:32.181 (Zigbee2MQTT) Joining new devices is disabled on the zigbee bridge
2021-02-11 08:29:32.232 (Zigbee2MQTT) New request: [2] devices_get({})
2021-02-11 08:29:32.232 (Zigbee2MQTT) MqttClient::publish zigbee2mqtt/bridge/config/devices/get ()

2021-02-11 08:29:32.352 (Zigbee2MQTT) New request: [3] groups_get({})
2021-02-11 08:29:32.352 (Zigbee2MQTT) MqttClient::publish zigbee2mqtt/bridge/config/groups ()
2021-02-11 08:29:32.412 (Zigbee2MQTT) Received groups list from bridge

2021-02-11 08:29:33.314 (Zigbee2MQTT) New request: [4] bridge_set_permitjoin("true")
2021-02-11 08:29:33.314 (Zigbee2MQTT) MqttClient::publish zigbee2mqtt/bridge/config/permit_join (true)
2021-02-11 08:29:33.730 (Zigbee2MQTT) Zigbee2mqtt log level is debug
2021-02-11 08:29:33.730 (Zigbee2MQTT) Joining new devices is enabled on the zigbee bridge
2021-02-11 08:29:34.282 (Zigbee2MQTT) New request: [5] bridge_set_permitjoin("true")
2021-02-11 08:29:34.282 (Zigbee2MQTT) MqttClient::publish zigbee2mqtt/bridge/config/permit_join (true)
2021-02-11 08:29:35.897 (Zigbee2MQTT) MqttClient::ping

2021-02-11 08:29:45.875 (Zigbee2MQTT) MqttClient::ping
2021-02-11 08:29:48.335 (Zigbee2MQTT) MqttClient::publish zigbee2mqtt/bridge/config/devices/get ()

2021-02-11 08:29:55.912 (Zigbee2MQTT) MqttClient::ping
2021-02-11 08:30:00.654 (Zigbee2MQTT) Received available devices list from bridge
2021-02-11 08:30:00.655 (Zigbee2MQTT) Coordinator address is 0x00124b00216000b9
2021-02-11 08:30:00.688 (Zigbee2MQTT) {"access": 7, "description": "Color temperature after cold power on of this light", "name": "color_temp_startup", "presets": [{"description": "Coolest temperature supported", "name": "coolest", "value": 150}, {"description": "Cool temperature (250 mireds / 4000 Kelvin)", "name": "cool", "value": 250}, {"description": "Neutral temperature (370 mireds / 2700 Kelvin)", "name": "neutral", "value": 370}, {"description": "Warm temperature (454 mireds / 2200 Kelvin)", "name": "warm", "value": 454}, {"description": "Warmest temperature supported", "name": "warmest", "value": 500}, {"description": "Restore previous color_temp on cold power on", "name": "previous", "value": 65535}], "property": "color_temp_startup", "type": "numeric", "unit": "mired", "value_max": 500, "value_min": 150}
2021-02-11 08:30:00.688 (Zigbee2MQTT) {"description": "Color of this light expressed as hue/saturation", "features": [{"access": 7, "name": "hue", "property": "hue", "type": "numeric"}, {"access": 7, "name": "saturation", "property": "saturation", "type": "numeric"}], "name": "color_hs", "property": "color", "type": "composite"}
2021-02-11 08:30:00.688 (Zigbee2MQTT) Creating domoticz device to handle "effect" key for device with ieeeAddr 0x0017880106a30bc3
2021-02-11 08:30:00.690 (Zigbee2MQTT) {"access": 7, "description": "Color temperature after cold power on of this light", "name": "color_temp_startup", "presets": [{"description": "Coolest temperature supported", "name": "coolest", "value": 150}, {"description": "Cool temperature (250 mireds / 4000 Kelvin)", "name": "cool", "value": 250}, {"description": "Neutral temperature (370 mireds / 2700 Kelvin)", "name": "neutral", "value": 370}, {"description": "Warm temperature (454 mireds / 2200 Kelvin)", "name": "warm", "value": 454}, {"description": "Warmest temperature supported", "name": "warmest", "value": 500}, {"description": "Restore previous color_temp on cold power on", "name": "previous", "value": 65535}], "property": "color_temp_startup", "type": "numeric", "unit": "mired", "value_max": 500, "value_min": 150}
2021-02-11 08:30:00.691 (Zigbee2MQTT) Creating domoticz device to handle "effect" key for device with ieeeAddr 0x00124b00222e14f5
2021-02-11 08:30:00.692 (Zigbee2MQTT) Creating domoticz device to handle "battery" key for device with ieeeAddr 0x0008890001d9f7e1
2021-02-11 08:30:00.693 (Zigbee2MQTT) Creating domoticz device to handle "action" key for device with ieeeAddr 0x0008890001d9f7e1
2021-02-11 08:30:00.696 (Zigbee2MQTT) Creating domoticz device to handle "linkquality" key for device with ieeeAddr 0x0008890001d9f7e1



2021-02-11 08:30:00.687 Error: (Zigbee2MQTT) 0x0017880106a30bc3: can not process numeric item "color_temp_startup"
2021-02-11 08:30:00.688 Error: (Zigbee2MQTT) 0x0017880106a30bc3: can not process feature type "composite"
2021-02-11 08:30:00.689 Error: Expected: myVar = Domoticz.Device(Name="myDevice", Unit=0, TypeName="", Type=0, Subtype=0, Switchtype=0, Image=0, Options={}, Used=1)
2021-02-11 08:30:00.689 Error: (Zigbee2MQTT) 'CDevice_init' failed 'TypeError':'an integer is required (got type NoneType)'.
2021-02-11 08:30:00.689 Error: Device creation failed, Device object is not associated with a plugin.
2021-02-11 08:30:00.690 Error: (Zigbee2MQTT) 0x00124b00222e14f5: can not process numeric item "color_temp_startup"
2021-02-11 08:30:00.691 Error: Expected: myVar = Domoticz.Device(Name="myDevice", Unit=0, TypeName="", Type=0, Subtype=0, Switchtype=0, Image=0, Options={}, Used=1)
2021-02-11 08:30:00.691 Error: (Zigbee2MQTT) 'CDevice_init' failed 'TypeError':'an integer is required (got type NoneType)'.
2021-02-11 08:30:00.691 Error: Device creation failed, Device object is not associated with a plugin.
2021-02-11 08:30:00.692 Error: Expected: myVar = Domoticz.Device(Name="myDevice", Unit=0, TypeName="", Type=0, Subtype=0, Switchtype=0, Image=0, Options={}, Used=1)
2021-02-11 08:30:00.692 Error: (Zigbee2MQTT) 'CDevice_init' failed 'TypeError':'an integer is required (got type NoneType)'.
2021-02-11 08:30:00.692 Error: Device creation failed, Device object is not associated with a plugin.
2021-02-11 08:30:00.695 Error: Expected: myVar = Domoticz.Device(Name="myDevice", Unit=0, TypeName="", Type=0, Subtype=0, Switchtype=0, Image=0, Options={}, Used=1)
2021-02-11 08:30:00.695 Error: (Zigbee2MQTT) 'CDevice_init' failed 'TypeError':'an integer is required (got type NoneType)'.
2021-02-11 08:30:00.695 Error: Device creation failed, Device object is not associated with a plugin.
2021-02-11 08:30:00.696 Error: Expected: myVar = Domoticz.Device(Name="myDevice", Unit=0, TypeName="", Type=0, Subtype=0, Switchtype=0, Image=0, Options={}, Used=1)
2021-02-11 08:30:00.696 Error: (Zigbee2MQTT) 'CDevice_init' failed 'TypeError':'an integer is required (got type NoneType)'.
2021-02-11 08:30:00.696 Error: Device creation failed, Device object is not associated with a plugin.
